src/environments/environment.prod.ts

```diff
@@ -1,3 +1,35 @@
+import { Profile } from '../app/auth/profiles/profiles.component';
+
 export const AppConfig = {
-  production: true
+  production: true,
+  clientId: '5e1ed08cff7f0de1d68d',
```

Comment:
Should we store these variables somewhere else, maybe in the form of `SECRETS` 

Else, it might be possible for others to get access to our client ids and possibly misuse them!